public/index.html

```diff
@@ -24,6 +27,14 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
```

Comment:
we need to put this into the data protection declaration, as well. maybe add it to the issue about imprint and DP, to not forget about it. Reason is, that google fonts is tracking the user as well